src/ui/sass/modules/_Results.scss

```diff
@@ -2,6 +2,7 @@
 
 $results-title-bar-background: var(--yxt-color-background-highlight) !default;
 $results-filters-background: white !default;
+$results-view-more-background: var(--yxt-color-background-highlight);
```

Comment:
Can we add !default if an AEB user wanted to override them

Reply:
sorry can't believe I forgot this